casr/tests/tests.rs

```diff
@@ -2440,15 +2440,16 @@ fn test_casr_cluster_c() {
         .parse::<u32>()
         .unwrap();
 
-    assert_eq!(after_cnt, 10, "After count mismatch.");
+    assert_eq!(after_cnt, 11, "After count mismatch.");
 
     // 2.casrep and 20.caserp without crashlines => no dedup
     // 3.casrep and 30.caserp with crashlines => dedup
-    // Thus, cluster (cl8) with 2.casrep has 2 casreps and others have 1 casrep
+    // Thus, cluster (cl7) with 2.casrep has 2 casreps and cl9 too
```

Comment:
(cl9)

Reply:
Both `cl7` and `cl9` have 2 casreps
`2.casrep` belongs to `cl7`